src/test/integration/fastapi/tests/test_fastapi.py

```diff
@@ -77,3 +94,80 @@ def test_requests_instrumentation(self):
         self.assertIsNotNone(children[0]["attributes"]["http.request.headers"])
         self.assertIsNotNone(children[0]["attributes"]["http.response.headers"])
         self.assertIsNotNone(children[0]["attributes"]["http.response.body"])
+
+    @skip_attr_size_limit
+    def test_large_span_attribute_size_default_max_size(self):
```

Comment:
I think that we run this file for every version of fastapi but I don't think that we need to run this test for every version of fastapi.